### ISSUE TITLE
PHP Warning on running psalm

### ISSUE THREAD

**Issue by descoder** [vimeo/psalm#11173]
When running in console

`vendor/bin/psalm --debug`

I get this warning:

```
Checking /app/vendor/bin/psalm
PHP Warning:  Undefined array key "extension" in /app/vendor/vimeo/psalm/src/Psalm/Checker/ProjectChecker.php on line 488

Warning: Undefined array key "extension" in /app/vendor/vimeo/psalm/src/Psalm/Checker/ProjectChecker.php on line 488
...
```

PHP version 8.1

I'm waiting for clear run.
